java/org/contikios/mrm/AreaViewer.java

```diff
@@ -2362,45 +2362,46 @@ public Collection<Element> getConfigXML() {
   @Override
   public boolean setConfigXML(Collection<Element> configXML, boolean visAvailable) {
     for (Element element : configXML) {
-      if (element.getName().equals("selected")) {
+      var name = element.getName();
+      if (name.equals("selected")) {
```

Comment:
Why not change to a switch instead, like in #697?

Reply:
I already had a 200+ line diff, so I saved that for later. There's also an empty branch in the middle of this that should be removed.

(Introducing the variable makes it possible for IntelliJ to make the conversion automatically though.)

Reply:
Introducing a local variable is also quite a line diff but ok.